plugins.json

```diff
@@ -11517,6 +11517,56 @@
     },
     "updated_at": "2021-01-15T11:07:27.026366282Z"
   },
+  {
+    "homepage_url": "https://github.com/mattermost/mattermost-plugin-memes",
+    "icon_data": "",
+    "download_url": "https://plugins-store.test.mattermost.com/release/mattermost-plugin-memes-v1.5.0.tar.gz",
+    "release_notes_url": "https://github.com/mattermost/mattermost-plugin-memes/releases/tag/v1.4.0",
```

Comment:
Is is `1.4.0` or `1.5.0` here and below?

Reply:
Good catch. There is something fishy with the release. Looking into it: https://community.mattermost.com/core/pl/cch8c5mnkbff9gsue4w3oibfbo

Reply:
Fixed